api/src/main/java/org/openmrs/Obs.java

```diff
@@ -1171,6 +1215,8 @@ public void setCreator(User creator) {
 	 * @see Auditable#setDateCreated(Date)
 	 */
 	@Override
+	@Access(AccessType.PROPERTY)
```

Comment:
This is already mapped in the superclass

Reply:
True.I have noticed.